content/influxdb/v2.4/security/tokens/create-token.md

```diff
@@ -136,14 +136,19 @@ influx auth create \
 
 #### Create an Operator token
 
-Create an Operator token to grant permissions to all resources in all organizations.
+Create an operator token to grant permissions to all resources in all organizations.
```

Comment:
I believe historically we capitalized Operator as a proper type of token? or maybe just bc it was shown like that in the UI? I don't have a strong lean on this one, other than keeping it consistent. We don't capitalize any other resources, although we are capitalizing All-Access, so there's that too

Reply:
Oh, you answered my question here. Exactly what I was wondering. "All-Access" and "Operator" is just convenient shorhand that describes the permissions given by the authorization. They're caps in the UI, but so is everything else.

Reply:
I'll caps them for now, but it kinda bugs me 🐛